wtforms_appengine/ndb.py

```diff
@@ -358,6 +360,9 @@ def convert_KeyProperty(self, model, prop, kwargs):
         else:
             return KeyPropertyField(**kwargs)
 
+    def convert__ClassKeyProperty(self, model, prop, kwargs):
```

Comment:
Ah, I see. Sorry for the confusion. If it helps, I can split the bugfix PR into a separate one.